app/models/bit_core/tool.rb

```diff
@@ -29,6 +29,10 @@ def last_position
       content_modules.order(:position).last.try(:position) || 0
     end
 
+    def slug_title
+      slug || title
```

Comment:
if a clinician wants to change the title of the tool without changing the url.  For instance, what I am currently working on with Sunnyside: they want the title to change from "MESSAGES" to "Contact Moderator".